src/actions/transformPlayerMatches.js

```diff
@@ -0,0 +1,16 @@
+export default function transformPlayerMatches(fields) {
```

Comment:
to avoid confusion (all transformations should be functions that operate on the input) i'd just make this file the inner function that operates on and returns the input.  You can put the wrapper function elsewhere (maybe in the actions file) and name it `generateTransformPlayerMatches` or something

Reply:
Sorry, I don't quite understand what you mean by this, currently I just followed how it was done for transformTrends.
What I understand from this is the wrapper function should be the actual transform function (as that is the code transforming the json response). But I only want to transform the json if the url query has an included account id (this information won't be available from the json response), hence the transformPlayerMatches function which is getting the query params from actions.

Reply:
yeah, I forgot about transformtrends, this is fine. Thanks!

The naming is a bit confusing but we can fix that later.